channels/chan_echolink.c

```diff
@@ -3363,7 +3388,8 @@ static int do_new_call(struct el_instance *instp, struct el_pvt *pvt, const char
 				ast_mutex_unlock(&el_db_lock);
 				return -1;
 			}
-			ast_raw_answer(chan);
+			fr.src = type;
+			ast_queue_frame(chan, &fr);
```

Comment:
While `ast_raw_answer()` appeared to work, we need to use `ast_queue_frame()` because we are in a channel driver here.